utils/config-utils.js

```diff
@@ -1,10 +1,30 @@
-// `CUTTLE_ENV` is needed to test prod specific files without actually putting the app in to
-// a production state, which requires an actual database connection
-const isProd =
-  process.env.CUTTLE_ENV !== undefined
-    ? process.env.CUTTLE_ENV === 'production'
-    : process.env.NODE_ENV === 'production';
-
-module.exports = {
-  isProd,
-};
+import devtools from '@vue/devtools'
+
+import { version } from '_/package.json';
+
+export function initCuttleGlobals(app) {
+  if (!window) {
+    console.error('No window object found');
+  }
```

Comment:
Defensive in case the window isn't available. This may not be required since we aren't SSR yet, I'd be ok removing it.